kani-compiler/src/codegen_cprover_gotoc/codegen/span.rs

```diff
@@ -53,6 +53,7 @@ impl<'tcx> GotocCtx<'tcx> {
         self.current_fn().mir().var_debug_info.iter().find(|info| match info.value {
             VarDebugInfoContents::Place(p) => p.local == *l && p.projection.len() == 0,
             VarDebugInfoContents::Const(_) => false,
+            VarDebugInfoContents::Composite { .. } => todo!(),
```

Comment:
Can you please create an issue and add more details there? I wonder if we should really crash in this case.